resources/phrase_lists/long_phrases.yml

```diff
@@ -0,0 +1,256 @@
+- "a 2015 study found that people posting duck face pictures are more likely to be associated with neuroticism and emotional instability."
+- "97 percent of all emails sent worldwide are spam."
+- "Eskimos use refrigerators to prevent food from freezing."
+- "there are about two million saunas in Finland, enough for the entire finnish population to take a sauna at the same time"
+- "you get twice as wet running in the rain versus walking."
+- "there is a city called Rome on every continent."
+- "Romans used to crush 4 million mollusk shells to create one pound of purple pigment. This is why the color Purple is associated with Royalty as only the Kings could afford that color"
+- "you get twice as wet running in the rain vs walking."
+- "in 1915, scam artist Charles Hatfield convinced the town of San Diego that he could create rainfall using a secret mix of chemicals. The city offered to pay him $10,000 if he could end their drought, and the 'result,' a few days later, was the town's worst flood of the 20th century."
+- "the sand dunes in the Sahara desert are known to have a phenomena known as \"singing sand\" - when conditions are right, the sand moving on dunes can make a humming sound."
+- "the Cuban missile crisis started when the United States noticed Cubans building football fields, because they knew Cubans preferred baseball over football. The fields were built for Soviet military camps in Cuba."
+- "in 2012 when Taylor Swift held a public vote for where she would visit and preform a free concert, 4chan users voted for a childrens school for the deaf and won the vote."
+- "from 1998 to 2003, the coach of the VfL Wolfsburg football team was named Wolfgang Wolf."
+- "around 62 percent of the 1000 most viewed YouTube videos are blocked in Germany."
+- "German chocolate cake is named after a guy named Sam German, not the country."
+- "The creature that kills the most people every year isn't snakes, sharks, or even other humans — it's the mosquito."
+- "There's an island in Japan you can visit that's inhabited only by friendly bunnies."
+- "There are actually more public libraries in the US than McDonalds"
+- "In 1518, a dancing plague took over the town of Strasbourg, France"
+- "Before the invention of color TV, 75% of people said they dreamed in black and white. Today, only 12% do."
+- "Wombat poop is cube-shaped."
+- "The British royal family is not allowed to play Monopoly."
+- "Paper bags aren't any better for the environment than plastic ones."
+- "There are lower-case numbers in addition to lower-case letters."
+- "Ancient Greeks came up with the idea of cyclops after they found a fossil of a mammoth, and had no idea what it was."
+- "Lego is the largest tire manufacturer in the world."
+- "The Hermitage Museum in St. Petersburg, Russia, has so many cats on its grounds that it employs a Press Secretary to the Cats."
```

Comment:
Does the verbalize St. to Saint or does it try to pronounce it like st

Reply:
Let's assume the worst ;) But we can fix this when we know specific error cases. I'd just adopt this for now.